doc/source/whatsnew/v2.1.0.rst

```diff
@@ -435,6 +435,7 @@ Groupby/resample/rolling
   grouped :class:`Series` or :class:`DataFrame` was a :class:`DatetimeIndex`, :class:`TimedeltaIndex`
   or :class:`PeriodIndex`, and the ``groupby`` method was given a function as its first argument,
   the function operated on the whole index rather than each element of the index. (:issue:`51979`)
+- Bug in :meth:`DataFrame.groupby` with column selection on the resulting groupby object not returning tuples when grouping by a list of a single element. (:issue:`53500`)
```

Comment:
We should also do the fix for Series.

Reply:
Sorry but can you give an example of doing column selection on Series groupby object? I thought this is a dataframe-only concept.

Reply:
df.groupby("a")["b"]